lib/rules/function-allowed-list/README.md

```diff
@@ -73,3 +73,46 @@ a {
     -moz-linear-gradient(45deg, blue, red);
 }
 ```
+
+## Optional secondary options
+
+### `exceptWithoutPropertyFallback`: `["array", "of", "unprefixed", /functions/, "/regex/"]|"function"|"/regex/"|/regex/`
+
+Disallow the matching functions when they are without a property fallback in the same declaration block.
+
+For example, with `["scale", "min", "/max/"]`.
```

Comment:
````suggestion
Given:

```js
["min", "/max/"]
```

For example, with `["scale", "min", "/max/"]` as the primary option.
````

Let's use this format to [be consistent with other rules](https://stylelint.io/user-guide/rules/selector-max-attribute/#ignoreattributes-regex-regex-non-regex).

Reply:
Sorry, I misunderstood. Thanks.